### ISSUE TITLE
Table body with missing data when markdown is padded

### ISSUE THREAD

**Issue by skyghis** [commonmark/commonmark-java#199]
Table rendering changed after version `0.15.2`.

For table (note spaces on the left of the table):

```markdown
  | Col 1 | Col 2 | Col 3 |
  |-------|-------|-------|
  | A1 | B1 | C1 |

```

With version `0.15.2` I get expected : 

```html
<table>
<thead>
<tr>
<th>Col 1</th>
<th>Col 2</th>
<th>Col 3</th>
</tr>
</thead>
<tbody>
<tr>
<td>A1</td>
<td>B1</td>
<td>C1</td>
</tr>
</tbody>
</table>

```

But with versions `0.16.1` and `0.17.0` I get a table with first column empty and last column missing: 

```html
<table>
<thead>
<tr>
<th>Col 1</th>
<th>Col 2</th>
<th>Col 3</th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td>A1</td>
<td>B1</td>
</tr>
</tbody>
</table>

```

Without spaces on the left of the Markdown table, the HTML is correct.

**Comment by robinst** [comment]
Yep, that's a bug! Will fix ASAP.

**Comment by robinst** [comment]
Fixed and released 0.17.1 with the fix: https://github.com/commonmark/commonmark-java/blob/main/CHANGELOG.md#0171---2021-02-03

Thanks for reporting!

**Comment by skyghis** [comment]
Thank you for this fast fix and release :+1:
